README.md

```diff
@@ -27,6 +27,11 @@ integration from the Jetpack Benchmark library.
 This sample compares how different ViewGroups
 (ConstraintLayout vs traditional layouts) affect the UI performance.
 
+## [MacrobenchmarkSample](MacrobenchmarkSample)
+
+This sample shows how to use the MacrobenchmarkSample library for testing aplication startup and runtime performance cases, such as scrolling a `RecyclerView` to measure jank.
+
```

Comment:
also add a line about the FTL integration, and link to the running-on-FTL doc from it.